gensim/models/keyedvectors.py

```diff
@@ -78,6 +78,7 @@
 from six import string_types, iteritems
 from six.moves import xrange
 from scipy import stats
+from keras.layers import Embedding
```

Comment:
Sure. Added a flag to check if Keras is installed or not. If not installed and function `get_embedding_layer()` called, then it raises an `ImportError`.